src/Raven.Studio/typescript/models/database/index/indexFieldOptions.ts

```diff
@@ -76,6 +80,9 @@ class indexFieldOptions {
             label: "Search",
             value: "Search"
         }];
+
+    static readonly IndexingWithSearchImplied: Array<valueAndLabelItem<indexingTypes, string>> =
```

Comment:
IIRC talked to use null as a value here, right?

Reply:
As discussed, currently leaving `analyzersNamesDictionary` & `IndexingWithSearchImplied` as is